tasks/main.yml

```diff
@@ -30,6 +30,7 @@
     name: ondemand
     state: stopped
     enabled: no
+  ignore_errors: yes
```

Comment:
instead of ignoring errors it would be better to gather [services_facts](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_facts_module.html) and add a `when` condition if `ondemand` is present, like
```
when: ansible_facts.services["ondemand.service"] exists
```